cmd/openshift-install/gather.go

```diff
@@ -145,8 +146,14 @@ func logGatherBootstrap(bootstrap string, port int, masters []string, directory
 	if err != nil && strings.Contains(err.Error(), "ssh: handshake failed: ssh: unable to authenticate") {
 		return errors.Wrap(err, "failed to create SSH client, ensure the private key is added to your authentication agent (ssh-agent) or specified with the --key parameter")
 	} else if err != nil {
+		if errno, ok := errors.Unwrap(err).(syscall.Errno); ok && errno == syscall.ECONNREFUSED {
+			return errors.Wrap(err, "failed to connect to the bootstrap machine")
+		} else if len(gatherBootstrapOpts.sshKeys) == 0 {
+			return errors.Wrap(err, "failed to create SSH client, ensure the proper ssh key is in your keyring or specify with --key")
+		}
```

Comment:
i think it's better use to the [`Is`](https://golang.org/pkg/errors/#Is) function introduced in go1.13, see `Overview` in https://golang.org/pkg/errors/#Unwrap

`github.com/pkg/errors` which we use in installer is transparent replacement for std lib `errors`.